### PR TITLE
created hexagon shape

### PR DESCRIPTION
**Pull request by JLambertazzo**

### CODE REVIEW

**Comment by ashutosh1919** [comment]
@JLambertazzo Thank you so much for raising PR. Let this PR stay open for 3-4 days. Since we have released the project yesterday, we want to stabilize the traffic as of now. I will start reviewing PRs starting from Monday (1st March) 💯

File: src/components/shapesdemos.tsx [hunk 1]
```diff
@@ -55,6 +55,15 @@ const polygoncardExample = `
 />
 `;
 
+const hexagonExample = `
+<Hexagon
+    height="150px"
+    width="175px"
+    zIndex={2}
+    color="linear-gradient(135deg, #7b2cbf, #3c096c)"
```
Comment (ashutosh1919) [inline src/components/shapesdemos.tsx:63]:
```suggestion
    color="linear-gradient(135deg, #c4b5fd, #8b5cf6)"
```

We are following [tailwind colors](https://tailwindcss.com/docs/customizing-colors) so it is good to stay consistent with the gradient theme which we have used in other components.

File: src/lib/shapes/hexagon.tsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  right?: string | string[];
+  bottom?: string | string[];
+  className?: string;
+  zIndex?: string;
+  breakpoints?: number[];
+  position?: string;
+};
+
+export const Hexagon = ({
+  size,
```
Comment (ashutosh1919) [inline src/lib/shapes/hexagon.tsx:21]:
```suggestion
```

`size` is not used anywhere in the shape so can you please remove it?

File: src/lib/shapes/hexagon.tsx [hunk 3]
```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import { BaseShape } from './base';
+
+type HexagonProps = {
+  size: string | string[];
```
Comment (ashutosh1919) [inline src/lib/shapes/hexagon.tsx:6]:
```suggestion
```

Please remove `size` from here as well.

**Review by ashutosh1919 — CHANGES_REQUESTED** [review]
@JLambertazzo , Thanks for contribution.

Below are some of the issues that you can fix before we can merge. Other than below issues, there is one more thing that you need to add.

In [Usage section of readme](https://github.com/ashutosh1919/react-awesome-shapes#usage), you need to add new shape name and you need to link it with the `hexagon.tsx`.

**Comment by JLambertazzo** [comment]
Hey sounds good! Changes have been made 👍

File: src/components/shapesdemos.tsx [hunk 4]
```diff
@@ -55,13 +56,22 @@ const polygoncardExample = `
 />
 `;
 
+
```
Comment (ashutosh1919) [inline src/components/shapesdemos.tsx:59]:
```suggestion
```

**Review by ashutosh1919 — CHANGES_REQUESTED** [review]

**Comment by ashutosh1919** [comment]
@all-contributors please add @JLambertazzo for code

**Comment by ashutosh1919** [comment]
Fixed #30

**Pull request merged**